### PR TITLE
Update some description about secrets

### PR DESCRIPTION
**Pull request by CaoShuFeng**
1. We can use RBAC/ABAC to control which users can access secrets.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2800)

### CODE REVIEW

File: docs/user-guide/secrets/index.md [hunk 1]
```diff
@@ -778,7 +778,8 @@ Pod level](#use-case-secret-visible-to-one-container-in-a-pod).
 
 ### Risks
 
- - In the API server secret data is stored as plaintext in etcd; therefore:
+ - If the API server runs with argument "--storage-media-type=application/json", secret data is stored as
+   plaintext in etcd; therefore:
```
Comment (mikedanese) [inline docs/user-guide/secrets/index.md]: I'm not sure that --storage-media-type makes a difference. It's still plain text, just surrounded by random bytes. Also technically it's not plaintext, it's base64 but that's quite irrelevant. Can we leave this part as is?

**Comment by mikedanese** [comment]
Second change looks good. Can we drop the first change?

**Comment by CaoShuFeng** [comment]
@mikedanese Done.

**Comment by mikedanese** [comment]
Tech LGTM

**Pull request merged**